### PR TITLE
Update contributing

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

**Review by core23 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -270,15 +220,13 @@ it SHOULD receive one if that version is a patch version.
...
+If the current major is `42`, a new `43.x` branch SHOULD be created from `42.x`.
 
 Also, the following files MUST be created/updated on the new stable branch:
 
  - `UPGRADE-43.x.md`, containing only the main title
  - `UPGRADE-43.0.md`, containing the upgrade notes fetched from the major PRs.
  - `CHANGELOG.md`, containing the changelog of the major PRs.
- - `composer.json`, the `branch-alias` MUST be changed to `43.x-dev`
 
 Push the new branch with a commit containing the modified files and "43.x-dev" as comment.
```
Comment (jordisala1991) [inline CONTRIBUTING.md:283]: I think this line should change or be removed too

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ and docs help a lot with that.
 - missing tests : Encourage people to do TDD by making clear a PR will not get merged
 if it lacks tests. Not everything is easy to test though, keep that in mind.
 - BC breaks : If there is a BC-break, ask for a deprecation system to be created instead,
-and make sure the `master` branch is used.
+and make sure the unstable branch is used.
```
Comment (OskarStark) [inline CONTRIBUTING.md:39]: We know the unstable branch pet project and could add it here to be more specific instead of talking about "unstable"

Comment (jordisala1991) [reply CONTRIBUTING.md:39]: But this is the global contributing guide , not the project specific

File: CONTRIBUTING.md (hunk 1 above)
Comment (VincentLanglet) [reply CONTRIBUTING.md:283]: Done

**Review by OskarStark — APPROVED** [review]

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**